src/main/java/org/sagebionetworks/web/client/widget/entity/EntityMetadata.java

```diff
@@ -49,8 +49,7 @@ public class EntityMetadata implements Presenter {
 	private PortalGinInjector ginInjector;
 	private ContainerItemCountWidget containerItemCountWidget;
 	private org.sagebionetworks.web.client.widget.entity.restriction.v2.RestrictionWidget restrictionWidgetV2;
-
-	boolean isShowingAnnotations;
+	private ActionMenuWidget actionMenu;
```

Comment:
Discovered than opening/closing annotations was bugged. Use the same process as TableEntityWidget panels, toggling the visibility using the action menu